frontend/src/app/(campgroundinfo)/campground/page.tsx

```diff
@@ -6,24 +6,40 @@ import Link from "next/link";
 import { getServerSession } from "next-auth";
 import { authOptions } from "@/app/api/auth/[...nextauth]/route";
 import getUserDashboard from "@/libs/getUserDashboard";
+import Announcement from "@/components/Announcement";
+import CreateAnnouncementCard from "@/components/CreateAnnouncementCard";
+import EditAnnouncementCard from "@/components/EditAnnouncementCard";
+import AnnouncementCard from "@/components/AnnouncementCard";
 
 export default async function Campground() {
   const campgrounds = getCampgrounds(50);
   const session = await getServerSession(authOptions);
   const profile = await getUserDashboard(session.user.token);
   if (!campgrounds) return null;
   return (
-    <main className="text-center p-5 mx-[8%]">
-      <div className="text-4xl font-bold m-10 text-left">Campground</div>
+    <main className="text-center p-5 mx-[4%] flex flex-row">
+      <div className="bg-cadetblue rounded-[20px] py-2 px-10 my-10 w-[35%] flex flex-col h-full">
+      <div className="flex flex-col my-[3%]">
```

Comment:
could these be it's own section?
ex. scrolling on appointment won't scroll campground and vice versa. Thx